test/libsolidity/gasTests/abiv2.sol

```diff
@@ -13,7 +13,7 @@ contract C {
     function f8(uint[32] memory, string[] memory, uint32, address) public returns (uint[] memory, uint16[] memory) {}
 }
 // ----
-// creation: 1160 + 1119000 = 1120160
```

Comment:
This is due to an additional assignment in the Yul code. The optimizer should take care of that later.